./configure --disable-tls broken

its broken by the following 2 independent bugs / commits:

```
$ ./configure --disable-tls
[...]
gcc -g -O2 -pipe -Wall -I.. -I..  -DHAVE_CONFIG_H -I/usr/include -g3 -DDEBUG -DDEBUG_ASSERT -DDEBUG_MEM -DDEBUG_DNS  -c botcmd.c
botcmd.c: In function ‘bot_nlinked’:
botcmd.c:726:53: error: ‘struct dcc_t’ has no member named ‘ssl’
  726 |   addbot(newbot, dcc[idx].nick, next, x, i, dcc[idx].ssl);
      |                                                     ^
make[1]: *** [Makefile:83: botcmd.o] Error 1
make[1]: Leaving directory '/home/michael/projects/eggdrop5/eggdrop/src'
make: *** [Makefile:251: debug] Error 2
```

broken since commit b8250a56b801ee7c1dd9f5d041f2c94d664958ab

```
gcc -fPIC -g -O2 -pipe -Wall -I. -I../../.. -I../../.. -I../../../src/mod  -DHAVE_CONFIG_H -I/usr/include -g3 -DDEBUG -DDEBUG_ASSERT -DDEBUG_MEM -DDEBUG_DNS  -DMAKING_MODS -c .././server.mod/server.c && mv -f server.o ../
In file included from .././server.mod/server.c:154:
.././server.mod/tclserv.c: In function ‘tcl_server’:
.././server.mod/tclserv.c:534:38: error: ‘struct server_list’ has no member named ‘ssl’
  534 |       snprintf(s, sizeof s, "%s%d", z->ssl ? "+" : "", z->port);
      |                                      ^~

```

broken since commit 881f5ee158c84926c85e1d01f73d3e466a79a73f

--- a/src/botcmd.c
+++ b/src/botcmd.c
@@ -723,7 +723,11 @@ static void bot_nlinked(int idx, char *par)
     putlog(LOG_BOTMSG, "*", "(%s) %s %s.", next, NET_LINKEDTO, newbot);
     x = '-';
   }
+#ifdef TLS
   addbot(newbot, dcc[idx].nick, next, x, i, dcc[idx].ssl);
+#else
+  addbot(newbot, dcc[idx].nick, next, x, i, 0);
+#endif
   check_tcl_link(newbot, next);
   u = get_user_by_handle(userlist, newbot);
   if (bot_flags(u) & BOT_REJECT) {

--- a/src/dcc.c
+++ b/src/dcc.c
@@ -244,7 +244,11 @@ static void bot_version(int idx, char *par)
   touch_laston(dcc[idx].user, "linked", now);
   dump_links(idx);
   dcc[idx].type = &DCC_BOT;
+#ifdef TLS
   addbot(dcc[idx].nick, dcc[idx].nick, botnetnick, '-', dcc[idx].u.bot->numver, dcc[idx].ssl);
+#else
+  addbot(dcc[idx].nick, dcc[idx].nick, botnetnick, '-', dcc[idx].u.bot->numver, 0);
+#endif
   check_tcl_link(dcc[idx].nick, botnetnick);
   egg_snprintf(x, sizeof x, "v %d", dcc[idx].u.bot->numver);
   bot_share(idx, x);

